src/flask/cli.py

```diff
@@ -1018,20 +1018,22 @@ def routes_command(sort: str, all_methods: bool) -> None:
         for rule in rules
     ]
 
-    headers = ("Endpoint", "Methods", "Rule")
+    headers = ("Sub Domain", "Endpoint", "Methods", "Rule")
     widths = (
+        max(len(rule.subdomain) for rule in rules),
```

Comment:
`subdomain` can be `None`, need to account for that.

Reply:
I did some edits for that, and ran pytest. 483 passed but 2 skipped. What does that mean. nothing fails after my edits.